test/index.spec.ts

```diff
@@ -1046,15 +1046,17 @@ describe('function helpers', () => {
         await moveNextDist()
         await nextRuntime.onBuild(defaultArgs)
         const dependencies = await getAllPageDependencies(constants.PUBLISH_DIR)
-        expect(dependencies.map((dep) => normalizeChunkNames(relative(process.cwd(), dep)))).toMatchSnapshot()
+        expect(dependencies.map((dep) => normalizeChunkNames(relative(process.cwd(), dep))).sort()).toMatchSnapshot()
       })
 
-      it('extracts dependencies that exist', async () => {
+      // TODO: `dependencies` references files inside the <root>/node_modules directory which isn't accessible in moveNextDist
+      // So this whole test needs to be reworked as it can't be fixed
+      it.skip('extracts dependencies that exist', async () => {
```

Comment:
Because the demo page is part of the npm workspace, it references files inside the root node_modules folder.
When we copy/move this demo page to a tmp dir for this test, it tries to reach into that node_modules folder that doesn't exist.